chingu-coders/Voyage2-Turtles-02
# pomodoro prototype

## Purpose
* Minimally functional standard clock with toggle for pomodoro clock

## Approach/Fixes
* Implements a core feature of the app

## Testing
* [check out this pull request](https://help.github.com/articles/checking-out-pull-requests-locally/).
* Confirm that standard clock shows correct time for your timezone.
* Try toggling between the two clocks
* let the pomodoro cycle run.
* listen for alarm.
* note that I purposefully made the pomodoro intervals short, so you could observe the functionality more quickly. Once we have "settings", we will define a default interval, and user can also select custom interval
* other note: I styled the pomodoro toggle button, but then it occurred to me that styling all hover, active and focus pseudo-selectors should probably be a single task for a single person, so as to enforce consistent styling.
